framework/src/actions/AutoCheckpointAction.C

```diff
@@ -25,23 +25,39 @@ AutoCheckpointAction::AutoCheckpointAction(const InputParameters & params) : Act
 void
 AutoCheckpointAction::act()
 {
-  if (_app.isUltimateMaster())
+  // if there's already a checkpoint object, we don't need to worry about creating a new
+  // checkpoint
+  const auto checkpoints = _app.getOutputWarehouse().getOutputs<Checkpoint>();
+  const auto num_checkpoints = checkpoints.size();
+
+  if (num_checkpoints > 1)
+  {
+    // Get most recently added Checkpoint object and error
+    checkpoints.back()->paramError(
+        "_type",
```

Comment:
Is this even a valid parameter to error on?

Reply:
This is the param that makes the most sense to error on, since `type` will always be `Checkpoint`. Otherwise, I could just change it to a MOOSE error.

Reply:
`_type` is a private parameter and you'll never get a line reference to it (doesn't exist in input). Just make it a moose error